db/migrations/02_add_posted_time_to_messages.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE peeps 
+ADD COLUMN posted VARCHAR(60);
```

Comment:
Would it be better for the database to store a timestamp rather than a varchar?

Reply:
Yes, it would, and I did initially, but was gettting issues with my tests, so I changed it. Your idea of using the strfttime in the html would have prevented these issues, and made it easier to adapt the code later